packages/tabs/package.json

```
@@ -68,6 +68,7 @@
     "@chakra-ui/utils": "1.8.0"
   },
   "devDependencies": {
+    "@chakra-ui/modal": "1.8.7",
```

Please remove this. I understand it's intended to be used in the stories but you can simply import from the `@chakra-ui/modal` without adding it. 

Since the `dist/` of the packages are hoisted to the root `node_modules`, there's nothing to worry about.

```suggestion
    "@chakra-ui/modal": "1.8.7",
```